src/neuroconv/tools/nwb_helpers/_dataset_configuration.py

```diff
@@ -114,7 +114,7 @@ def get_default_dataset_io_configurations(
                     continue
 
                 dataset_io_configuration = DatasetIOConfigurationClass.from_neurodata_object(
-                    neurodata_object=column, field_name="data"
+                    neurodata_object=column, dataset_name="data"
```

Comment:
Minor style change, mentioned in the CHANGELOG

Since it doesn't affect the recommended, or common way of using these brand-new features I'm bypassing the need for a deprecation alert

Reply:
I think it makes sense.